.mypy.ini

```diff
@@ -0,0 +1,22 @@
+[mypy]
```

Comment:
Add instructions/comment about this in DEVELOPMENT.md ?

Reply:
I don't think there's much to say, it shouldn't come up for the average developer, and when it does, mypy prints instructions / it's pretty standard to add ignored libraries when they don't have type stubs.